### DISCUSSION TITLE
Multiple sync workers run together

### DISCUSSION THREAD

**Discussion by strahe** [filecoin-project/lotus#7773] [Build and Sync] [answered]
![图片](https://user-images.githubusercontent.com/7932644/145740212-767f3da8-d6da-4bdc-88ba-b14c9efaa1dc.png)

![图片](https://user-images.githubusercontent.com/7932644/145740246-d3f3102a-c678-4291-8d2e-2cc2473a2dda.png)

Is this expected? why is this happening?

**Comment by de-authority** [comment]
I think it's expected. it's pretty normal.  Do you get any problems?

**Comment by SgtCoin** [comment, marked as answer]
Archiving due to age.
